Website/src/rendering/src/components/Navigation/Breadcrumb.tsx

```diff
@@ -1,73 +1,39 @@
 import React from 'react';
 import { ComponentProps } from 'lib/component-props';
 
-function formatBreadcrumb(text: string) {
-  text = decodeURIComponent(text);
-  return text;
-}
-
-type contextItem = {
-  url: { path: string };
-  name: string;
-  pageTitle: { value: string };
+type Breadcrumb = {
```

Comment:
Could you please create a ShopBreadcrumb component and revert this one? This one is using a Sitecore GraphQL query results and we might want to keep it as is for the rest of the website. Thanks